src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -708,4 +708,11 @@ Valid from:</comment>
 2 - package source url
 3 - Log Message</comment>
   </data>
+  <data name="VerifyCertTrustOfflineWhileRevocationModeOffline" xml:space="preserve">
+    <value>The revocation function was unable to check revocation because the certificate is not available in the cached certificate revocation list and certificateRevocationMode has been set to offline. For more information, visit https://aka.ms/certificateRevocationMode.</value>
+  </data>
+  <data name="VerifyCertTrustOfflineWhileRevocationModeOnline" xml:space="preserve">
+    <value>The revocation function was unable to check revocation because the revocation server could not be reached. For more information, visit https://aka.ms/certificateRevocationMode.</value>
```

Comment:
Same here...is the revocation function the right language?

Reply:
yes it is because it the CertGetCertificateChain function that does returns this error.